src/Type/Accessory/AccessoryNumericStringType.php

```diff
@@ -296,6 +296,11 @@ public function isLiteralString(): TrinaryLogic
 		return TrinaryLogic::createMaybe();
 	}
 
+	public function isLowercaseString(): TrinaryLogic
+	{
+		return TrinaryLogic::createMaybe();
```

Comment:
I'd say yes.

Reply:
"10E4" is a numeric string not lowercase

Reply:
Yes that is the reason I kept maybe